source/modulo_core/src/translators/parameter_translators.cpp

```diff
@@ -184,15 +188,18 @@ std::shared_ptr<state_representation::ParameterInterface> read_parameter(const r
   throw exceptions::ParameterTranslationException("Parameter " + parameter.get_name() + " could not be read!");
 }
 
-std::shared_ptr<state_representation::ParameterInterface> read_parameter_const(
-    const rclcpp::Parameter& ros_parameter,
-    const std::shared_ptr<const state_representation::ParameterInterface>& parameter
+std::shared_ptr<ParameterInterface> read_parameter_const(
+    const rclcpp::Parameter& ros_parameter, const std::shared_ptr<const ParameterInterface>& parameter
 ) {
   if (ros_parameter.get_name() != parameter->get_name()) {
     throw exceptions::ParameterTranslationException(
         "The ROS parameter " + ros_parameter.get_name()
             + " to be read does not have the same name as the reference parameter " + parameter->get_name());
   }
+  if (ros_parameter.get_type() == rclcpp::PARAMETER_NOT_SET) {
+    return make_shared_parameter_interface(
```

Comment:
This function was a nice way to resolve the empty copy / construction for NOT SET 👍